### PR TITLE
add GPLv3+ header to all lua scripts

### PR DESCRIPTION
**Pull request by jrb0001**

### CODE REVIEW

File: lua/stacktraceplus.lua [hunk 1]
```diff
@@ -1,3 +1,11 @@
+--[[
```
Comment (pchote) [inline lua/stacktraceplus.lua]: We are using this from another project (https://github.com/ignacio/StackTracePlus IIRC), which is MIT license.

File: lua/sandbox.lua [hunk 2]
```diff
@@ -1,3 +1,11 @@
+--[[
+   Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
```
Comment (pchote) [inline lua/sandbox.lua]: This is another MIT licenced module (see text just below) that we are using unmodified.

File: lua/stacktraceplus.lua (hunk 1 above)
Comment (jrb0001) [inline lua/stacktraceplus.lua]: I copied the license text from it's repository.

File: lua/sandbox.lua (hunk 2 above)
Comment (jrb0001) [inline lua/sandbox.lua]: I removed the header completely because the license text follows below.

**Review by pchote — APPROVED** [review]
LGTM, but please squash the three commits into 1.

**Comment by jrb0001** [comment]
Done but you could have used that squash merge button of github...

File: mods/ts/maps/sunstroke/sunstroke.lua [hunk 3]
```diff
@@ -1,3 +1,11 @@
+--[[
+   Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
 Tick = function()
```
Comment (atlimit8) [inline mods/ts/maps/sunstroke/sunstroke.lua]: Does this script even meet the threshold of originality? This script may not even be copyrightable.

Comment (jrb0001) [inline mods/ts/maps/sunstroke/sunstroke.lua]: Does it have any effect if we put a license header on something which is not copyrightable?

**Review by obrakmann — APPROVED** [review]
👍

**Pull request merged**